doc/dependencies.Rmd

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Do we want to check this in? How will we keep this up to date? Do you imagine these changes will also flow into the roxygen vignette roclet? 

Should we also create an md version for easy previewing on github?

Reply:
I don't know about whether to check in or not, maybe we should not, but the benefit would be users would not have to build vignettes themselves. You would have to update it with `devtools::build_vignette()`.

Intermediates like `.md` are challenging to deal with without special casing rmarkdown vignettes and rendering them separately. It is somewhat questionable how much utility they have particularly now that we have pkgdown to view devel vignettes.

Reply:
Good point re: pkgdown. Some of my vignette whining may no longer relevant. If I have a balky vignette (takes a non-trivial amount of time to render, calls internet, needs auth, etc.), I would turn it into an article now.

That just transfers some whining from devtools to pkgdown 😅 but I still think lowers the overall amount of pain. Why? Because pkgdown is designed to make the built product (the site) constantly available, whereas the official vignette workflow is much more tied to an official build/release event.